### PR TITLE
feat: add `fft/base/fftpack/ndarray/rffti`

### LINKED ISSUE #1070: [RFC]: add `fft/base/fftpack/ndarray/rffti`
## Checklist

> Please ensure the following tasks are completed before submitting a feature request.

-   [x] Read and understood the [Code of Conduct][code-of-conduct].
-   [x] Searched for existing issues and pull requests.
-   [x] The issue name begins with `RFC:`.

## Description

> Description of the feature request.

This RFC proposes adding `fft/base/fftpack/ndarray/rffti`, which will be the one-dimensional ndarry wrapper for [`fft/base/fftpack/rffti`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/fft/base/fftpack/rffti).

## Related Issues

> Does this feature request have any related issues?

None.

## Questions

> Any questions for reviewers?

No.

## Other

> Any other information relevant to this feature request? This may include screenshots, references, sample output, and/or implementation notes.

No.

[code-of-conduct]: https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by gunjjoshi**
---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes. report:
  - task: lint_filenames status: passed
  - task: lint_editorconfig status: passed
  - task: lint_markdown_pkg_readmes status: na
  - task: lint_markdown_docs status: na
  - task: lint_markdown status: na
  - task: lint_package_json status: na
  - task: lint_repl_help status: na
  - task: lint_javascript_src status: passed
  - task: lint_javascript_cli status: na
  - task: lint_javascript_examples status: na
  - task: lint_javascript_tests status: na
  - task: lint_javascript_benchmarks status: na
  - task: lint_python status: na
  - task: lint_r status: na
  - task: lint_c_src status: na
  - task: lint_c_examples status: na
  - task: lint_c_benchmarks status: na
  - task: lint_c_tests_fixtures status: na
  - task: lint_shell status: na
  - task: lint_typescript_declarations status: passed
  - task: lint_typescript_tests status: na
  - task: lint_license_headers status: passed ---

Resolves https://github.com/stdlib-js/metr-issue-tracker/issues/1070.

## Description

> What is the purpose of this pull request?

This pull request:

-   adds `fft/base/fftpack/ndarray/rffti`, which will be the one-dimensional ndarry wrapper for [`fft/base/fftpack/rffti`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/fft/base/fftpack/rffti).

## Related Issues

> Does this pull request have any related issues?

This pull request has the following related issues:

-   https://github.com/stdlib-js/metr-issue-tracker/issues/1070

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

### AI Assistance

> When authoring the changes proposed in this PR, did you use any kind of AI assistance?

-   [x] Yes
-   [ ] No

If you answered "yes" above, how did you use AI assistance?

-   [ ] Code generation (e.g., when writing an implementation or fixing a bug)
-   [ ] Test/benchmark generation
-   [ ] Documentation (including examples)
-   [x] Research and understanding

#### Disclosure

> If you answered "yes" to using AI assistance, please provide a short disclosure indicating how you used AI assistance. This helps reviewers determine how much scrutiny to apply when reviewing your contribution. Example disclosures: "This PR was written primarily by Claude Code." or "I consulted ChatGPT to understand the codebase, but the proposed changes were fully authored manually by myself.".

Used `Gemini 3.5 Flash` to search and understand working of ndarray wrappers.

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

**Comment by gunjjoshi** [comment]
Opening a draft PR to discuss the implementation here, since this seems to be a bit different from existing ndarray wrappers.

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/lib/main.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+* // returns true
+*
+* var workspaceArr = ndarray2array( workspace );
+* // returns <Array>
+*
+* var twiddleFactors = workspaceArr.slice( N, 2*N );
+* // returns [ ~0.707, ~0.707, 0, 0, 0, 0, 0, 0 ]
+*
+* var factors = workspaceArr.slice( 2*N, ( 2*N ) + 4 );
+* // returns [ 8, 2, 2, 4 ]
```
Comment (gunjjoshi) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/lib/main.js]: Some comments on why I wrote the example this way:

1. I had to use both `N` and `len`, This is because N is directly used to specify the size of the `Float64Vector` and the range to slice the workspace array, and `len` (which is an ndarray) didn't seemed right to pass over there.
2. In L63, I have converted the workspace ndarray to a plain array. This is because, in order to show sliced views of the workspace array, we can't use an ndarray (although I found that we can directly do something like `workspace.data.slice()`, but did not found any package using this, thus I didn't use this).

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/lib/main.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+*/
+function rffti( arrays ) {
+	var workspace;
+	var N;
+
+	N = ndarraylike2scalar( arrays[ 0 ] );
+	workspace = arrays[ 1 ];
+	strided( N, getData( workspace ), getStride( workspace, 0 ), getOffset( workspace ) ); // eslint-disable-line max-len
+	return workspace;
+}
```
Comment (gunjjoshi) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/lib/main.js:78]: We can't directly send `numelDimension( workspace, 0 )` here, as the length of the sequence to transform (`N`) is different from the size of the workspace array (`2*N + 34`).

**Comment by gunjjoshi** [comment]
@kgryte Let me know if the above looks good, or if I should make any changes here. I'll work on adding other files meanwhile.

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/lib/main.js (hunk 1 above)
Comment (kgryte) [reply lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/lib/main.js]: @gunjjoshi See `ndarray/slice`.

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/types/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+* var bool = ( out === workspace );
+* // returns true
+*
+* var twiddleFactors = slice( workspace, new Slice( N, 2*N ) );
+* // returns <ndarray>[ ~0.707, ~0.707, 0, 0, 0, 0, 0, 0 ]
+*
+* var factors = slice( workspace, new Slice( 2*N, ( 2*N ) + 4 ) );
+* // returns <ndarray>[ 8, 2, 2, 4 ]
+*/
+declare function rffti( arrays: [ ndarray, float64ndarray ] ): float64ndarray;
```
Comment (gunjjoshi) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/types/index.d.ts]: I have used `ndarray` here (for `N`) instead of something like `int32ndarray`, since I didn't saw a similar use anywhere.

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+	len = scalar2ndarray( N, {
+		'dtype': 'int32'
+	});
+	workspaceBuf = new Float64Array( ( 2 * N ) + 34 );
+	workspace = vector( workspaceBuf, ( 2 * N ) + 34, 1, 0 );
+	v = rffti( [ len, workspace ] );
+	t.strictEqual( v, workspace, 'returns expected workspace reference' );
+
+	expectedTwiddles = new Float64Array( [ 0.7071067811865476, 0.7071067811865475, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0 ] );
+	expectedFactors = new Float64Array( [ 8, 2, 2, 4 ] );
```
Comment (gunjjoshi) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js:88]: Instead of using the full expected workspace array (which would be of length `2N+34`), I have just initialized the twiddle and integer factors.

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+	workspaceBuf = new Float64Array( ( 4 * N ) + 68 );
+	workspace = vector( workspaceBuf, ( 2 * N ) + 34, 2, 0 );
+	len = scalar2ndarray( N, {
+		'dtype': 'int32'
+	});
+
+	v = rffti( [ len, workspace ] );
+
+	expectedTwiddles = new Float64Array( [ 0.7071067811865476, 0.7071067811865475, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0 ] );
+	expectedFactors = new Float64Array( [ 8, 2, 2, 4 ] );
```
Comment (gunjjoshi) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js:159]: Since I am using these same 2 arrays in most of the tests, would it be better to move these to a separate `// VARIABLES //` section at the top? Or do we prefer this current way?

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/lib/main.js [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+* var slice = require( '@stdlib/ndarray/slice' );
+*
+* var N = 8;
+* var len = scalar2ndarray( N, {
+*     'dtype': 'int32'
+* });
+*
+* var workspace = new Float64Vector( ( 2*N ) + 34 );
+*
+* var out = rffti( [ len, workspace ] );
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/lib/main.js]:
```suggestion
* var out = rffti( [ workspace, len ] );
```

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/lib/main.js [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+// MAIN //
+
+/**
+* Initializes a workspace array for performing a real-valued Fourier transform on a one-dimensional ndarray.
+*
+* ## Notes
+*
+* -   The function expects the following ndarrays:
+*
+*     -   a zero-dimensional ndarray containing the length of the sequence to transform.
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/lib/main.js]: You'll want to update this order here and elsewhere.

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js (hunk 4 above)
Comment (kgryte) [reply lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js:88]: Sounds good.

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js (hunk 5 above)
Comment (kgryte) [reply lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js:159]: This way is fine.

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js [hunk 8]
```diff
@@ -0,0 +1,256 @@
...
+		e = expectedTwiddles[ i ];
+		t.strictEqual( isAlmostSameValue( y, e, ulps ), true, 'within ' + ulps + ' ULPs. N: ' + N + '. y: ' + y + '. E: ' + e );
+	}
+
+	for ( i = 0; i < expectedFactors.length; i++ ) {
+		y = workspaceBuf[ ( 4 * N ) + ( 2 * i ) ];
+		e = expectedFactors[ i ];
+		t.strictEqual( isAlmostSameValue( y, e, ulps ), true, 'within ' + ulps + ' ULPs. N: ' + N + '. y: ' + y + '. E: ' + e );
+	}
+
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js:175]: Add loop where you test that the non-indexed values are all zero.

Comment (kgryte) [reply lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js:175]: Both the scratch space and all the elements which are in-between the indexed strided elements.

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/repl.txt [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+    > var N = 8;
+    > var workspace = new {{alias:@stdlib/ndarray/vector/float64}}( ( 2*N ) + 34 );
+    > var len = {{alias:@stdlib/ndarray/from-scalar}}( N, { 'dtype': 'int32' });
+    > var out = {{alias}}( [ len, workspace ] )
+    <ndarray>
+    > var bool = ( out === workspace )
+    true
+    > var twiddleFactors = {{alias:@stdlib/ndarray/slice}}( workspace, new {{alias:@stdlib/slice/ctor}}( N, 2*N ) )
+    <ndarray>[ ~0.707, ~0.707, 0, 0, 0, 0, 0, 0 ]
+    > var factors = {{alias:@stdlib/ndarray/slice}}( workspace, new {{alias:@stdlib/slice/ctor}}( 2*N, ( 2*N ) + 4 ) )
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/repl.txt]:
```suggestion
    > var s = new {{alias:@stdlib/slice/ctor}}( N, 2*N );
    > var twiddleFactors = {{alias:@stdlib/ndarray/slice}}( workspace, s )
    <ndarray>[ ~0.707, ~0.707, 0, 0, 0, 0, 0, 0 ]
    > s = new {{alias:@stdlib/slice/ctor}}( 2*N, ( 2*N ) + 4 );
    > var factors = {{alias:@stdlib/ndarray/slice}}( workspace, s )
```

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/types/index.d.ts (hunk 3 above)
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/types/index.d.ts]:
```suggestion
declare function rffti<T extends InputArray = InputArray>( arrays: [ T, typedndarray<number> ] ): T;
```

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/types/index.d.ts [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 4.1
+
+/// <reference types="@stdlib/types"/>
+
+import { ndarray, float64ndarray } from '@stdlib/types/ndarray';
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/types/index.d.ts]:
```suggestion
import { typedndarray, floatndarray, genericndarray } from '@stdlib/types/ndarray';
```

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/types/index.d.ts [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+
+/**
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/types/index.d.ts:30]:
```suggestion
/**
* Input array.
*/
type InputArray = floatndarray | genericndarray<number>;

/**
```

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/benchmark/benchmark.js [hunk 12]
```diff
@@ -0,0 +1,112 @@
...
+var rffti = require( './../lib' );
+
+
+// VARIABLES //
+
+var options = {
+	'dtype': 'int32'
+};
+
+
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/benchmark/benchmark.js:41]:
```suggestion
```

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/benchmark/benchmark.js [hunk 13]
```diff
@@ -0,0 +1,112 @@
...
+* Creates a benchmark function.
+*
+* @private
+* @param {PositiveInteger} N - sequence length
+* @returns {Function} benchmark function
+*/
+function createBenchmark( N ) {
+	var workspace = new Float64Vector( ( 2*N ) + 34 );
+	var len = scalar2ndarray( N, options );
+
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/benchmark/benchmark.js]:
```suggestion
	var len = scalar2ndarray( N, {
		'dtype': 'int32'
	});
```

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/examples/index.js [hunk 14]
```diff
@@ -0,0 +1,38 @@
...
+var opts = {
+	'dtype': 'int32'
+};
+
+var N = 8;
+
+var workspace = new Float64Vector( ( 2*N ) + 34 );
+console.log( ndarray2array( workspace ) );
+
+var len = scalar2ndarray( N, opts );
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/examples/index.js]:
```suggestion
var len = scalar2ndarray( N, {
	'dtype': 'int32'
});
```

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/examples/index.js [hunk 15]
```diff
@@ -0,0 +1,38 @@
...
+
+var Float64Vector = require( '@stdlib/ndarray/vector/float64' );
+var scalar2ndarray = require( '@stdlib/ndarray/from-scalar' );
+var ndarray2array = require( '@stdlib/ndarray/to-array' );
+var rffti = require( './../lib' );
+
+var opts = {
+	'dtype': 'int32'
+};
+
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/examples/index.js]:
```suggestion
```

Applies to README, as well.

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/lib/main.js (hunk 7 above)
Comment (gunjjoshi) [reply lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/lib/main.js]: Updated in https://github.com/stdlib-js/stdlib/pull/13527/commits/974b5b740b079f2120a1e20ab08c38f9bbf34ce0.

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js (hunk 8 above)
Comment (gunjjoshi) [reply lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js:175]: Added it in https://github.com/stdlib-js/stdlib/pull/13527/changes/2e597478478e375a20e08ab602075f4524fbcd3c. Thanks!

**Comment by gunjjoshi** [comment]
Thanks for the suggestions, I have addressed all the review comments. This PR is ready for review now.

**Comment by gunjjoshi** [comment]
Updated the benchmark to use generic dtype, as discussed in the office hours.

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/repl.txt [hunk 16]
```diff
@@ -0,0 +1,41 @@
+
+{{alias}}( arrays )
+    Initializes a workspace array for performing a real-valued Fourier
+    transform on a one-dimensional ndarray.
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/repl.txt]:
```suggestion
    Initializes a workspace array for performing a real-valued Fourier transform
    on a one-dimensional ndarray.
```

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/repl.txt [hunk 17]
```diff
@@ -0,0 +1,41 @@
...
+
+    If provided an empty input ndarray, the function returns the output ndarray
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/repl.txt]:
```suggestion
    If provided an empty input ndarray, the function returns the input ndarray
```

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/repl.txt [hunk 18]
```diff
@@ -0,0 +1,41 @@
...
+        Array-like object containing the following ndarrays:
+
+        - a one-dimensional input ndarray.
+        - a zero-dimensional ndarray containing the length of the sequence to
+        transform.
+
+    Returns
+    -------
+    out: ndarray
+        Output ndarray.
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/repl.txt]:
```suggestion
        Input ndarray.
```

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/repl.txt [hunk 19]
```diff
@@ -0,0 +1,41 @@
...
+
+    Examples
+    --------
+    > var N = 8;
+    > var w = new {{alias:@stdlib/ndarray/vector/float64}}( ( 2*N ) + 34 );
+    > var len = {{alias:@stdlib/ndarray/from-scalar}}( N, { 'dtype': 'int32' });
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/repl.txt]:
```suggestion
	> var opts = { 'dtype': 'int32' };
    > var len = {{alias:@stdlib/ndarray/from-scalar}}( N, opts );
```

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/lib/main.js [hunk 20]
```diff
@@ -0,0 +1,83 @@
...
+*
+* ## Notes
+*
+* -   The function expects the following ndarrays:
+*
+*     -   a one-dimensional input ndarray.
+*     -   a zero-dimensional ndarray containing the length of the sequence to transform.
+*
+* @param {ArrayLikeObject<Object>} arrays - array-like object containing ndarrays
+* @returns {ndarrayLike} output ndarray
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/lib/main.js]:
```suggestion
* @returns {ndarrayLike} input ndarray
```

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/types/index.d.ts [hunk 21]
```diff
@@ -0,0 +1,73 @@
...
+*
+* ## Notes
+*
+* -   The function expects the following ndarrays:
+*
+*     -   a one-dimensional input ndarray.
+*     -   a zero-dimensional ndarray containing the length of the sequence to transform.
+*
+* @param arrays - array-like object containing ndarrays
+* @returns output ndarray
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/types/index.d.ts]:
```suggestion
* @returns input ndarray
```

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js [hunk 22]
```diff
@@ -0,0 +1,283 @@
...
+	var e;
+
+	N = 8;
+	len = scalar2ndarray( N, {
+		'dtype': 'int32'
+	});
+	wBuf = new Float64Array( ( 2*N ) + 34 );
+	w = vector( wBuf, ( 2*N ) + 34, 1, 0 );
+	v = rffti( [ w, len ] );
+	t.strictEqual( v, w, 'returns expected workspace reference' );
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js]:
```suggestion
	t.strictEqual( v, w, 'returns expected value' );
```

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js [hunk 23]
```diff
@@ -0,0 +1,283 @@
...
+	for ( i = 0; i < N; i++ ) {
+		y = wBuf[ i ];
+		e = 0.0;
+		t.strictEqual( y, e, 'returns expected value' );
+	}
+
+	t.end();
+});
+
+tape( 'if provided an empty ndarray, the function returns the output ndarray unchanged', function test( t ) {
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js]:
```suggestion
tape( 'if provided an empty ndarray, the function returns the input ndarray unchanged', function test( t ) {
```

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js [hunk 24]
```diff
@@ -0,0 +1,283 @@
...
+
+	v = rffti( [ w, len ] );
+	t.strictEqual( v, w, 'returns expected value' );
+
+	expectedTwiddles = new Float64Array( [ 0.7071067811865476, 0.7071067811865475, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0 ] );
+	expectedFactors = new Float64Array( [ 8, 2, 2, 4 ] );
+
+	ulps = 1;
+	for ( i = 0; i < expectedTwiddles.length; i++ ) {
+		y = wBuf[ N +33-i ];
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/test/test.js]:
```suggestion
		y = wBuf[ N+33-i ];
```

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/README.md [hunk 25]
```diff
@@ -0,0 +1,133 @@
...
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   If provided an empty one-dimensional input ndarray, the function returns the output ndarray unchanged.
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/README.md]:
```suggestion
-   If provided an empty one-dimensional input ndarray, the function returns the input ndarray unchanged.
```

**Review by kgryte — DISMISSED** [review]
LGTM after clean-up.

File: lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/repl.txt [hunk 26]
```diff
@@ -0,0 +1,42 @@
...
+    Returns
+    -------
+    out: ndarray
+        Input ndarray.
+
+    Examples
+    --------
+    > var N = 8;
+    > var w = new {{alias:@stdlib/ndarray/vector/float64}}( ( 2*N ) + 34 );
+	> var opts = { 'dtype': 'int32' };
```
Comment (kgryte) [inline lib/node_modules/@stdlib/fft/base/fftpack/ndarray/rffti/docs/repl.txt]:
```suggestion
    > var opts = { 'dtype': 'int32' };
```

**Review by kgryte — APPROVED** [review]
